manifests/plugin/python.pp

```diff
@@ -47,6 +49,8 @@
     globals  => $globals,
   }
 
+  ensure_packages([$python_package], { ensure => $python_package_ensure })
```

Comment:
we should probably wrap this into a $manage_python condition. There are distributions where 'python' defaults to python 3, others default to version 2. Do we need to catch that here as well?

Reply:
Collectd is always built against exactly one python. It is the `python` package untill we have a counter example.

Happy to add a manage_python boolean. It will help people who install python in another funky way.